### ISSUE TITLE
copyfiles is missing

### ISSUE THREAD

**Issue by vporton** [aragon/aragon-cli#1144]
## 🐛 Bug Report

`node_modules/@aragon/toolkit/node_modules/lie/package.json` contains a copyfiles command.

This command is not available on Ubuntu 19.10. So it does not work.

`@aragon/toolkit@0.0.2`

Need to add a dependency.

### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/aragon-cli/blob/master/CONTRIBUTING.md#ways-to-contribute)?

Yes.

**Comment by macor161** [comment]
Hi @vporton, thanks for submitting this. Can you share more info about how I can reproduce the problem?

**Comment by vporton** [comment]
```sh
git clone git@github.com:aragon/aragon-react-boilerplate.git
cd aragon-react-boilerplate
npm i @aragon/cli
npm i @aragon/ui
npm i debug
npm start
```

```
npm ERR! placeholder-app-name-frontend@1.0.0 sync-assets: `copy-aragon-ui-assets ../dist && copyfiles './public/**/*' ../dist`
```

**Comment by 0x6431346e** [comment]
> This command is not available on Ubuntu 19.10. So it does not work.

This command is exposed by a `npm` module: https://github.com/aragon/aragon-react-boilerplate/blob/master/app/package.json#L20

Please use the `npx create-aragon-app` utility instead of cloning the repository, or, if you rather use the cloned repo, run `npm i` in both the `root` and the `app` folder.
